backend/src/integrations/food-trust/food-trust-api.service.ts

```diff
@@ -0,0 +1,243 @@
+import axios, { AxiosRequestConfig} from 'axios';
+import {lotCommissionXML, serialAndPalletCommissionXML, lotObservationXML, serialAndPalletObservationXML, transformationXML, aggregationXML} from './xmlHelpers'
+
+let expiration = 0;
+let iftToken = "";
+const serviceTokenAPI = 'https://iam.cloud.ibm.com/identity/token';
+const iftAPI = 'https://sandbox.transparentsupply.ibm.com/ift/api/'
```

Comment:
Could you make this an env variable so it's configurable, because an in-production IFT instance will be running on a different URL.

```suggestion
const iftAPI = process.env.IFT_URL
if (iftAPI === undefined) {
    throw new Error("IFT_URL wasn't set in the environment!");
}
```

Reply:
done